queue/driver_group_mgo.go

```diff
@@ -396,16 +396,16 @@ func (d *mgoGroupDriver) Next(ctx context.Context) amboy.Job {
 		},
 	}
 
+	timeLimits := bson.M{}
+	now := time.Now()
 	if d.opts.CheckWaitUntil {
-		qd = bson.M{
-			"$and": []bson.M{
-				qd,
-				{"$or": []bson.M{
-					{"time_info.wait_until": bson.M{"$lte": time.Now()}},
-					{"time_info.wait_until": bson.M{"$exists": false}}},
```

Comment:
Did you deliberately remove this part of the $or?

Reply:
yes, I believe that it is no longer required.